girder/utility/plugin_utilities.py

```diff
@@ -122,7 +122,7 @@ def loadPlugins(plugins, root, appconf, apiRoot=None, buildDag=True):
     return root, appconf, apiRoot
 
 
-def getToposortedPlugins(plugins=None, ignoreMissing=False, keys=('dependencies',)):
```

Comment:
Is there a reason `logNew` has to exist at all? Can we just always add the info log message?